frontend/tests/unit/tests/services/query-service-test.js

```diff
@@ -131,10 +163,9 @@ describe('QueryService', function() {
       $httpBackend.flush();
     }
 
-    beforeEach(inject(function(_$httpBackend_, _PathHelper_, _$rootScope_) {
+    beforeEach(inject(function(_$httpBackend_, _PathHelper_) {
```

Comment:
Identifier '_$httpBackend_' is not in camel case.<br>Identifier '_PathHelper_' is not in camel case.